docs/docs/quickstart/lido.mdx

```diff
@@ -21,7 +21,7 @@ If you are interested in becoming a Lido node operator, make sure to read the [N
 
 :::info
 
-Lido CSM is **[currently live and permissionless on the Holesky and Mainnet testnet](https://operatorportal.lido.fi/modules/community-staking-module)**.
+Lido CSM is **[currently live and permissionless on the Holesky and Mainnet testnets](https://operatorportal.lido.fi/modules/community-staking-module)**.
```

Comment:
Mainnet Testnets?

Reply:
to refer to both the Holesky and Mainnet testnets